composer.json

```diff
@@ -24,10 +24,11 @@
         "doctrine/orm": "^2.6",
         "jdorn/sql-formatter": "^1.1",
         "mikey179/vfsStream": "^1.6",
-        "phpstan/phpstan": "^0.10",
-        "phpstan/phpstan-deprecation-rules": "^0.10",
-        "phpstan/phpstan-phpunit": "^0.10",
-        "phpstan/phpstan-strict-rules": "^0.10",
+        "phpstan/phpstan": "^0.11",
+        "phpstan/phpstan-deprecation-rules": "^0.11",
+        "phpstan/phpstan-phpunit": "^0.11",
+        "phpstan/phpstan-strict-rules": "^0.11",
+        "phpstan/phpstan-symfony": "^0.11.6",
```

Comment:
If you think 473d1bb16bf13355546e2936c3f8e7997983ae1b could be extracted in a separate PR that's something I could have people try to do at an Hacktoberfest event next week. Might help with making that PR smaller. Please tell me if there are any other parts you think would be easy to extract in separate PRs.

Reply:
PHPStan has been a major thing in this refactoring... i did a lot of iterations in fixing PHPStan complains... not sure how proceed here

Reply:
Ok so it's probably too big for that Hacktoberfest event then, nevermind.